h/accounts/models.py

```diff
@@ -67,6 +67,16 @@ class User(Base):
                           nullable=False,
                           unique=True)
 
+    display_name = sa.Column('display_name', sa.UnicodeText())
```

Comment:
No need to duplicate the column name. `sa.Column(sa.UnicodeText())` would be fine.